entregable_APIS_Daniel_Adam

adamgdaniel opened the pull request:
Entregable APIs

# PULL REQUEST

Si has llegado hasta aquí es que ya estás preparado para hacer tu PR:

Antes de continuar... asegúrate de lo siguiente:

Pon una `x` en las cajas siguientes para asegurar que has seguido el proceso:

- [x] He probado mi código y funciona en local
- [x] He validado que mi código es correcto a niveles de formato
- [x] Me he asegurado de no haber comiteado o filtrado ninguna contraseña
- [x] No he copiado mi código porque soy una persona muy responsable :-)

## Opcional:

Pega una meme que exprese como te has sentido durante tu trabajo, no es obligatorio pero es divertido :-P

adamgdaniel committed:
entregable_API_adamgdaniel

Entregable APIs

adamgdaniel committed:
entregable_API_adamgdaniel

Entregable APIs

adamgdaniel committed:
Merge branch 'entregable_APIS_adamgdaniel' of https://github.com/a10pepo/EDEM_MDA2526 into entregable_APIS_adamgdaniel

adamgdaniel committed:
COPY removal from dockerfile

Elimino el COPY del dockerfile ya que he creado un volume en el Compose

Marcolapietro commented on ALUMNOS/MDAA/DANIEL_ADAM/APIS/main.py:
```diff
@@ -0,0 +1,133 @@
+
+def post_tweet(content):
+    try:
+        load_dotenv()
```
load_dotenv() no debería llamarse dentro de cada post. Cárgalo una vez al inicio y valida credenciales antes.

Marcolapietro commented on ALUMNOS/MDAA/DANIEL_ADAM/APIS/main.py:
```diff
@@ -0,0 +1,133 @@
+
+# Funciones de la API de terremotos
+
+def monitor_terremotos():
```
Estás ignorando el error real. Capturas el error con en "e" pero luego no lo imprimes

Marcolapietro reviewed (dismissed):
La aplicación es interesante y combinar un menú interactivo, consumo de una API externa y publicación automática demuestra buen entendimiento de un flujo completo. El código es fácil de seguir y la idea de generar alertas accionables está muy bien. Buen trabajo!

adamgdaniel committed:
changes_requested_fix

Moved load_dotenv() from inside the function to the start of the script. Added error information to the print when API connection fails.

adamgdaniel reviewed (commented):
Todas las modificaciones arregladas y commiteadas.

a10pepo reviewed (changes requested):
Requesting changes because this PR mixes the APIS deliverable with changes in another assignment folder. The APIS submission must stay contained within the student's APIS folder only. Please split the work into separate branches and PRs.

adamgdaniel committed:
Delete ALUMNOS/MDAA/DANIEL_ADAM/GCP_ALMACENAMIENTO/terraform.tfvars

Elimino el file del proyecto de GCP almacenamiento que estaba en esta rama. La PR ya solo modifica archivos de una sola carpeta.
